src/strategies/dimo-balance-of-or-votes-with-overrides/index.ts

```diff
@@ -0,0 +1,90 @@
+import { BigNumberish } from '@ethersproject/bignumber';
+import { formatUnits } from '@ethersproject/units';
+import { getAddress } from '@ethersproject/address';
+import { Multicaller } from '../../utils';
+import { strategy as erc20VotesWithOverrideStrategy } from '../erc20-votes-with-override';
+
+export const author = '0xMaharishi';
+export const version = '0.1.0';
+export const nilAddress = '0x0000000000000000000000000000000000000000';
+
+const abi = [
+  'function delegates(address account) external view returns (address)',
+  'function getVotes(address account) external view returns (uint256)',
+  'function totalSupply() public view returns (uint256)',
+  'function balanceOf(address account) public view returns (uint256)'
+];
+
+interface Options {
+  address: string;
+  decimals: number;
+  includeSnapshotDelegations?: boolean;
+  delegationSpace?: string;
+}
+
+/*
+  Based on the `erc20-votes-with-override` strategy, with fallback to balanceof
+*/
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options: Options,
+  snapshot
+) {
+  const blockTag = typeof snapshot === 'number' ? snapshot : 'latest';
+  const checksummedAddresses = addresses.map((address) => getAddress(address));
+  const scores: Record<string, number> = await erc20VotesWithOverrideStrategy(
```

Comment:
Hi @zer0stars We stopped accepting strategies that depend on other addresses, (erc20-votes-with-override depend on other addresses) Refer this https://github.com/snapshot-labs/snapshot-strategies#code

Make it independent such that it doesn't call this strategy or doesn't depend on other address's voting power